tests/MongoDB.Driver.Tests/Linq/MongoQueryableTests.cs

```diff
@@ -1136,6 +1134,35 @@ public void Select_followed_by_group()
                 "{ $group: { _id: '$First', Stuff: { $push: { Id: '$Id', Second: '$Second' } } } }");
         }
 
+
+        [SkippableFact]
+        public void Select_with_using_top_level_expression_input_parameters_inside_children_levels()
+        {
+            RequireServer.Check().Supports(Feature.AggregateArrayFilter);
```

Comment:
I haven't been able to figure how the AggregateArrayFilter feature is required for this test.

Reply:
I see my confusion, there are two very similarly named features which represent completely different things:

    Feature.AggregateArrayFilter
    Feature.ArrayFilters

That's a bummer.

Reply:
Maybe I'm wrong.
When I launched this build on evergreen, I caught the following error:
`MongoDB.Driver.MongoCommandException : Command aggregate failed: exception: invalid operator '$filter'.`
Since this filter has been used inside `aggregate` command:

        aggregate([{ "$project" : { "Result" : { "$filter" : { "input" : "$G", "as" : "g1", "cond" ...

, I've decided that it's `Aggregate filter`. 
As I see from this link, `arrayFilter` it's a little bit another thing: https://docs.mongodb.com/manual/reference/operator/update/positional-filtered/. Could you please clarify this point? :)

Reply:
My point was simply that we have two Features with very similar names and I was confused and mixed them up. At first I didn't understand why you added AggregateArrayFilters when we already had ArrayFilters.